graphviz2drawio/mx/MxGraph.py

```
@@ -167,4 +171,4 @@ def __str__(self) -> str:
         return self.value()
 
     def __repr__(self) -> str:
-        return self.value()
+        return self.value()
```

Ruff (0.8.0)

No newline at end of file

Add trailing newline

(W292)